foxpuppet/browser/window.py

```diff
@@ -2,12 +2,14 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.windows import Windows
-
 from selenium.webdriver.common.by import By
 
+from foxpuppet.windows import BaseWindow
+from foxpuppet.windows import new_window_is_opened
+from foxpuppet.windows import WindowManager
```

Comment:
Multiple imports from the same module should be written as `from module import thing1, thing2`